index.html

```diff
@@ -262,7 +262,9 @@ <h2>Best Practices and Recommendations</h2>
         
         <p class="advisement" id="ltli-valid-content-req"><a href="#ltli-valid-content-req" class="self">&#x200B;</a>Specifications SHOULD require content authors use <a>valid</a> language tags.</p>
         <p class="advisement" id="ltli-validator-req"><a href="#ltli-validator-req" class="self">&#x200B;</a>Content validators SHOULD check if content uses <a>valid</a> language tags where feasible.</p>
-		<p class="advisement" id="ltli-language-information-in-uris-req"><a href="#ltli-language-information-in-uris-req" class="self">&#x200B;</a>Applications (e.g. in the realm of RDF) that provide language information as part of URIs SHOULD use [[BCP47]].</p>
+		<p class="advisement" id="ltli-language-information-in-uris-req"><a href="#ltli-language-information-in-uris-req" class="self">&#x200B;</a>Applications that provide language information as part of URIs (e.g. in the realm of RDF) SHOULD use [[BCP47]].</p>
+		
+		    <p>Currently, URIs expressing language information often use values from parts of ISO 639. This leads to situations in which there are ambiguities about what the proper value should be, e.g. for German "ger" from ISO 639-1 or "de" from ISO 639-2. By using BCP 47 and its language sub tag registry, such ambiguities can be avoided, e.g. for German, the registry contains only “de”.</p>
```

Comment:
ISO 639-1 == alpha2 and 639-2 == alpha3, so the example is backwards. 

Use <code> around language tags (remove quotes).

Reply:
Thanks, fixed.